core/src/helpers.rs

```diff
@@ -78,6 +78,42 @@ pub const fn ethers_block_number_to_starknet_block_id(
     }
 }
 
+/// # Errors
+///
+/// TODO: Will return `KakarotClientError`..
```

Comment:
why is this a todo?

Reply:
Made `decode_eth_call_return` based off `decode_execute_at_address_return` which had this TODO. I guess it can be removed for both functions now that they return `KakarotClientError`?